pilot/pkg/features/experimental.go

```diff
@@ -190,10 +190,6 @@ var (
 	PassthroughTargetPort = env.Register("ENABLE_RESOLUTION_NONE_TARGET_PORT", true,
 		"If enabled, targetPort will be supported for resolution=NONE ServiceEntry").Get()
 
-	PersistOldestWinsHeuristicForVirtualServiceHostMatching = env.Register("PERSIST_OLDEST_FIRST_HEURISTIC_FOR_VIRTUAL_SERVICE_HOST_MATCHING", false,
```

Comment:
It is still false, make it true at first?

Reply:
It's inverted from most flags - making it true gives the legacy behavior

Reply:
OH, got it